Spigot-API-Patches/0317-Banner-custom-name-API.patch

```diff
@@ -0,0 +1,36 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Swedz <me@swedz.net>
+Date: Fri, 18 Jun 2021 01:20:44 -0400
+Subject: [PATCH] Banner custom name API
+
+
+diff --git a/src/main/java/org/bukkit/block/Banner.java b/src/main/java/org/bukkit/block/Banner.java
+index e6eb3c04b9fa6cd232f95dd5851d365234972efd..5f9a3710825cc84ab1276d7e26707ebdfcbdd308 100644
+--- a/src/main/java/org/bukkit/block/Banner.java
++++ b/src/main/java/org/bukkit/block/Banner.java
+@@ -83,4 +83,25 @@ public interface Banner extends TileState {
```

Comment:
This needs to extend the bukkit Nameable interface.

Reply:
Just did the Nameable interface bit. Not sure how to go about swapping this to 1.17

Reply:
I also just realized I screwed up the commit. I had copied some code from the Beacon implementation for Nameable and didn't remove the unused imports. Fixing that now